docs/contributing.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Will this be concatenated with CONTRIBUTING.md in the root?

Reply:
`CONTRIBUTING.md` in the root has been combined with this.